File: root/programs/VB/Samples/WebApp_sample/ProjectX_sample/Startup.vb
'**********************************************************************************
'* テンプレート
'**********************************************************************************

' 以下のLicenseに従い、このProjectをTemplateとして使用可能です。Release時にCopyright表示してSublicenseして下さい。
' https://github.com/OpenTouryoProject/OpenTouryo/blob/master/license/LicenseForTemplates.txt

'**********************************************************************************
'* クラス名        ：OwinStartup
'* クラス日本語名  ：OwinStartup
'*
'* 作成日時        ：－
'* 作成者          ：－
'* 更新履歴        ：－
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'**********************************************************************************

Imports Owin
Imports Microsoft.Owin

Imports System.Web.Optimization
Imports System.Web.Routing

<Assembly: OwinStartup(GetType(ProjectX_sample.Startup))>

Namespace ProjectX_sample
    Public Class Startup
        Public Sub Configuration(app As IAppBuilder)
            ' アプリケーションの設定方法の詳細については、http://go.microsoft.com/fwlink/?LinkID=316888 を参照してください

            ' アプリケーションのスタートアップで実行するコードです

            ' URLルーティングの登録
            RouteConfig.RegisterRoutes(RouteTable.Routes)
            ' バンドル＆ミニフィケーションの登録
            BundleConfig.RegisterBundles(BundleTable.Bundles)
        End Sub
    End Class
End Namespace

File: root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Startup.cs
//**********************************************************************************
//* テンプレート
//**********************************************************************************

// サンプル中のテンプレートなので、必要に応じて流用して下さい。

//**********************************************************************************
//* クラス名        ：OwinStartup
//* クラス日本語名  ：OwinStartup
//*
//* 作成日時        ：－
//* 作成者          ：－
//* 更新履歴        ：－
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//**********************************************************************************

using Owin;
using Microsoft.Owin;

using System.Web.Optimization;
using System.Web.Routing;

[assembly: OwinStartup(typeof(ProjectX_sample.Startup))]

namespace ProjectX_sample
{
    public class Startup
    {
        public void Configuration(IAppBuilder app)
        {
            // アプリケーションの設定方法の詳細については、http://go.microsoft.com/fwlink/?LinkID=316888 を参照してください

            // アプリケーションのスタートアップで実行するコードです

            // URLルーティングの登録
            RouteConfig.RegisterRoutes(RouteTable.Routes);
            // バンドル＆ミニフィケーションの登録
            BundleConfig.RegisterBundles(BundleTable.Bundles);
        }
    }
}
